docs/adding_packages/dependencies.md

```diff
@@ -5,90 +5,24 @@ from handling "vendored" dependencies to what versions should be used.
 
 <!-- toc -->
 ## Contents
-
-  * [List Dependencies](#list-dependencies)
-    * [Optional Requirements](#optional-requirements)
-    * [Build Requirements](#build-requirements)
-  * [Accessing Dependencies](#accessing-dependencies)
-    * [Handling Requirement's Options](#handling-requirements-options)
-    * [Verifying Dependency's Version](#verifying-dependencys-version)
-    * [Passing Requirement's info to `build()`](#passing-requirements-info-to-build)
-    * [Overriding the provided properties from the consumer](#overriding-the-provided-properties-from-the-consumer)
+  * [Handling Requirement's Options](#handling-requirements-options)
   * [Adherence to Build Service](#adherence-to-build-service)
     * [Version Ranges](#version-ranges)
-      * [Adding Version Ranges](#adding-version-ranges)
+    * [Adding Version Ranges](#adding-version-ranges)
   * [Handling "internal" dependencies](#handling-internal-dependencies)<!-- endToc -->
 
-## List Dependencies
-
-Since all ConanCenterIndex recipes are to build and/or package projects they are exclusively done in [`conanfile.py`](https://docs.conan.io/1/reference/conanfile.html). This offers a few
-ways to add requirements. The most common way is [requirements](https://docs.conan.io/1/reference/conanfile/methods.html#requirements):
-
-```py
-    def requirements(self):
-        self.requires("fmt/9.1.0")
-```
-
-> **Note**: With Conan 2.0, you'll also need to pay attention to new properties like the `transitive_header` attributed which is
-> needed when a project include a dependencies header files in its public headers.
-
-When a project supports a range of version of a dependency, it's generally advised to pick the **most recent available in ConanCenter**.
-This helps ensure there are fewer conflicts with other, up to-date, recipes that share the same requirement.
-
-### Optional Requirements
-
-Many projects support enabling certain features by adding dependencies. In ConanCenterIndex this is done by adding an option, see
-[naming recommendation](conanfile_attributes.md#recommended-names), which should be set to match the upstream project's by default.
-
-```py
-class ExampleConan(ConanFile):
-    options = {
-        "with_zlib": [True, False], # Possible values
-    }
-    default_options = {
-        "with_zlib": True, # Should match upstream's CMakeLists.txt `option(...)`
-    }
-
-    def requirements(self):
-        if self.options.with_zlib:
-            self.requires("zlib/1.2.13")
-```
 
-If a dependency was added (or removed) with a release, then the `if` condition could check [`self.version`](https://docs.conan.io/1/reference/conanfile/attributes.html#version). Another common case is
-`self.settings.os` dependant requirements which need to be added for certain plaforms.
 
-### Build Requirements
```

Comment:
Tool requirements is well covered by Conan docs:

- https://docs.conan.io/2/reference/conanfile/methods/build_requirements.html
- https://docs.conan.io/2/examples/graph/tool_requires/using_protobuf.html#examples-graph-tool-requires-protobuf
-